### ISSUE TITLE
Consider using GNU style params for e2e test script

### ISSUE THREAD

**Issue by vnugent** [openshift/origin-metrics#87]
I was hoping to be able to skip the image build by specifying `--skipBuild=false` but the build ran anyway.

**Comment by mwringe** [comment]
`--skipBuild=false` would mean that you don't want to skip the build.

`--skipBuild=true` would mean that you want to skip the build.

Is the build properly being skipped with `--skipBuild=true`?

**Comment by vnugent** [comment]
I intended to say the build will still happen with `--skipBuild=true`

Output from recent run
```
# ./e2e-tests.sh --debug --timeout=420 --skipBuild=true
...
(B[m[INFO] Starting Origin-Metric end-to-end test(B[m
(B[m[INFO] (B[m
(B[m[INFO] Settings:(B[m
(B[m[INFO] Base Directory: /tmp/origin-metrics(B[m
(B[m[INFO] ================================================================================(B[m
(B[m[INFO] (B[m
(B[m[INFO] (B[m
(B[m[INFO] Building new images(B[m
Building image openshift/origin-metrics-hawkular-metrics:latest
Sending build context to Docker daemon 35.84 kB
Sending build context to Docker daemon 35.84 kB

Step 0 : FROM jboss/wildfly:10.0.0.Final
 ---> 7281c168bc1b
Step 1 : MAINTAINER Hawkular Metrics <hawkular-dev@lists.jboss.org>
 ---> Running in 8f13748bc281
...


```

**Comment by mwringe** [comment]
Ah, sorry, I get the issue now. I have only just been using `--skipBuild` and not `--skipBuild=true`. I can update it so that it works for both

**Comment by bbguimaraes** [comment]
Fixed on #171.
